tests/fast/CycleAndLock.txt

```diff
@@ -25,4 +25,4 @@ testTitle=Clogged
     machinesToKill=10
     machinesToLeave=3
     reboot=true
-    testDuration=30.0
\ No newline at end of file
+    testDuration=30.0
```

Comment:
what is the change here?

Reply:
I think previously there was no newline at the end of the file. I think my editor did that